app/models/experimentation/experiment.rb

```diff
@@ -0,0 +1,19 @@
+module Experimentation
+  class Experiment < ActiveRecord::Base
+    has_many :treatment_buckets
+
+    validates :lookup_name, presence: true, uniqueness: true
+    validates :state, presence: true
+    validates :experiment_type, presence: true
+
+    enum state: {
```

Comment:
We probably also want to have a `completed` state so we can easily grab finished experiments for data collection.